### DISCUSSION TITLE
Completely Lost

### DISCUSSION THREAD

**Discussion by WashCaps33** [riffnshred/nhl-led-scoreboard#249] [General]
Started the project.  Did the setup as shown, but completely lost when trying to get into the Pi.  The password “raspberry” is not working. I need massive help.  I stink at this stuff.

**Comment by pjockey** [comment]
I guess the first question would be, how are you trying to access your RPi? Do you have a screen and keyboard connected to the  Pi? If I know this I can perhaps guide you along.

**Reply by WashCaps33** [reply]
It would be awesome if you could help.  I’m dying to get it up and running.  Anyhow, I’m using SSH to my laptop.  I know it’s connected to my wifi because I located it’s IP address.  But, when using Putty and using the IP, it asks for user and password.  Use “pi” and “raspberry” and get an Access Denied message.

**Comment by pjockey** [comment]
I'm not a putty user, but I've had the same errors using a terminal as well and it is frustrating. For me, it came down to the port I was using for the connection.  

I'm assuming you don the following;
1. https://howchoo.com/g/ndy1zte2yjn/how-to-set-up-wifi-on-your-raspberry-pi-without-ethernet
2. https://howchoo.com/g/ote0ywmzywj/how-to-enable-ssh-on-raspbian-without-a-screen

The second one may be the problem. Take the sd card out of the Pi and back into your computer/adapter and redo 1 and 2 and then try again. I remember seeing my IP address of the Pi but couldn't access as the password or SSID entered in #1 was missed typed. And the reason I could see that was the file name, wpa_supplicant.conf was also miss typed. So, redo 1 and 2 and see if that helps.

**Comment by riffnshred** [comment]
Hey bud, which tutorial or instructions are you following?

**Reply by WashCaps33** [reply]
Following what is listed in #7.   Had Raspian Buster lite on a micro SD card, set up wpa_supplicant file and the blank SSH file.  Had no trouble with it connecting to my wifi.  But when it came to accessing the Pi itself, nothing but Access Denied.  I made no changes to any passwords, so I can’t figure out why the default password won’t work.  I ended up formatting my micro SD card and I’m stumped.  I’m way in over my head here


https://howchoo.com/pi/diy-raspberry-pi-nhl-scoreboard-led-panel

**Reply by riffnshred** [reply]
Forget that tutorial and read my documentation. There are 2 method to install the software. I recommend you to go with Method 1.

**Reply by WashCaps33** [reply]
Well good news...actually got in.  Doing the sudo apt-get ups grade now.  I’m sure I’ll need more help.  I’m sorry to be bugging you guys.

**Reply by pjockey** [reply]
good to hear.

**Reply by riffnshred** [reply]
if you follow the steps we provided, you should get up and running without any issues.

**Reply by WashCaps33** [reply]
Working on it now.  Real quick.  Nothing will appear on the LED board until everything has been done, correct?  I did all the hardware requirements, including the switch, but have yet to see anything appear on it.

**Reply by riffnshred** [reply]
exactly. if you use Method 1 with the OS Image that comes with the software pre-installed, it will make you do the Hello World, that will be you test to see if your screen and your setup works

**Reply by WashCaps33** [reply]
I did see that message once....tried to do it again and failed completely.  I’m all screwed up on step 5 here

https://github.com/riffnshred/nhl-led-scoreboard/wiki/Step-by-step-guide.

**Reply by riffnshred** [reply]
make sure you are at the root of the folder, `cd nhl-led-scoreboard`. than run ./nhl_setup.

**Reply by WashCaps33** [reply]
Shouldn’t it reboot in the root already?  Going back to it now and I’m right back at the beginning within the command area.

**Reply by riffnshred** [reply]
no, on reboot it brings you to the user directory. If you follow each step precisely when you get to step 5 you are still in the root of the nhl-led-scoreboard as you just installed it. 

If you are confused, do 

`cd`
then 
`cd nhl-led-scoreboard`  

that will bring you there. 

EDIT: when I say the root of the nhl-led-scoreboard, I'm not talking about the root of the raspberry pi but the nhl-led-scoreboard software it self.

**Reply by riffnshred** [reply]
Like I said, I recommand doing Method 1 instead of the step by step because it takes care of a lot of things for you, like remove most of the terminal navigation stuff etc....

**Reply by WashCaps33** [reply]
Doing method 1 and 10.0.0.168:9001 will not work in my browser.

I’ll gladly pay you for the help if you can get me through this.  I know I’m being a massive pain.

**Reply by WashCaps33** [reply]
I can’t get to the damn Supervisor.  Been trying all day. About to bail on the project entirely.

**Reply by riffnshred** [reply]
Which raspberry pi are you using and are you using the bonnet or the HAT to connect the panel ?

**Reply by riffnshred** [reply]
If you are at the setup to check supervisor, you are about there. Just to confirm, you are still following Method 1 ?

**Reply by WashCaps33** [reply]
I am using method 1. I do as instructed,
Yet when I jump over to my browser is is unable to connect.  Using what I established my Pi’s IP address and :9001.

And I am using a Raspberry Pi Zero WH and the Adafruit RGB Bonnet

**Reply by riffnshred** [reply]
Thank for the info. It's impossible for me to know what causes your issues and it can be caused by factors unrelated to my software (can be network-related or hardware related etc....) 

First thing.... put that thing aside and start over another time. This is a DIY project and not everyone makes it the first time (even myself when I started that project) and even the simple things can make the installation fail. Working on this while being salty and stuff will raise the chance of doing something wrong reduce the chance of finding what does not work. 

Second... Since you were able to get to the point of setting up the config while you were following Method 2 (the Step by step guide) I would recommend starting over and go that route again. When you get the nhl_setup app and configuring your scoreboard, make sure you are at the base level of the nhl-led-scoreboard directory when you run the `./nhl_setup` command.  When it will ask you if you want to do the quick config (select one team), say yes, pick your team and save.  Then take your time with the Testing and Optimization phase. this is where optimize your raspberry pi as much as possible and test to see if your hardware works and also its the first time you run the scoreboard. Only when you are able to run the scoreboard using the command at the end of the step, you can move one to step 7 which is when you install and run Supervisor.

Lastly.... What can help make things run smoothly is using a Raspberry Pi 3B+ or 3A+ . if you do, make sure you electrically Isolate the bottom of the Bonnet as it can touch and short something. 

It's been known for awhile now that the Pi Zero would not stand the evolution of this project and Im about to roll it out completely as its just slow. My recommendation is the Pi 3A+ but make sure you Isolate the bottom of the Bonnet as it touches the USB port.

**Reply by WashCaps33** [reply]
I am supposed to be using the IP address of the Pi to run Supervisor, correct?  It’s right there where I think I am going wrong.

**Reply by riffnshred** [reply]
yup. the ip address you are using to connect to it on the terminal with port :9001.

If you are in a terminal right now, go to the nhl-led-scoreboard directory and run this command

`sudo python3 src/main.py --led-gpio-mapping=adafruit-hat --led-brightness=50`

If the scoreboard is installed properly and you have configured it, it should run properly. If not, it will spit out errors and that will tell you whats wrong.

**Reply by WashCaps33** [reply]
Fires right up perfectly.  It’s something about that :9001 port thing that is derailing it.  Got all the way to the end and can’t quite get over the hump.  I want it to automatically boot up when I turn it on.

**Reply by WashCaps33** [reply]
maybe seeing this will help determine what I’m doing wrong.

![858EF61E-6BD0-45CA-949A-5A356FF0356B](https://user-images.githubusercontent.com/80552501/111088933-ea751280-84ff-11eb-9cae-baa8114befc9.jpeg)

**Reply by riffnshred** [reply]
thats good news then. Alright. open up the supervisor.conf file 

`sudo nano /etc/supervisor/supervisord.conf`  

and show me whats in there

**Reply by riffnshred** [reply]
ok 2 things

take out the --led-slowdown-gpio=2 form the command, 

then in the directory remove the brakets around `pi`. try that

**Reply by riffnshred** [reply]
OH also change the `--led-gpio-mapping=adafruit-hat-pwm` for `--led-gpio-mapping=adafruit-hat`

**Reply by WashCaps33** [reply]
Ok those are done.  Here is a photo of the step before that.  Where I feel like I may be screwing something up.

![576AAAD2-193F-4377-B7E1-0263D71DDAB6](https://user-images.githubusercontent.com/80552501/111089772-a5eb7600-8503-11eb-95c3-5fbc7eff29de.jpeg)

**Reply by riffnshred** [reply]
all good there. reboot it now

`sudo reboot -n`

You should be able to see supervisor in your browser

**Reply by WashCaps33** [reply]
Site can’t be reached
10.0.0.168 refused to connect 

Assuming that mean it’s my internet?

**Reply by riffnshred** [reply]
is that the IP you are using to connect to your scoreboard from a terminal ?  I believe it will start with  192.168 and not 10.0. 

Otherwise, download advance IP scanner and scan for you raspberry pi, it will show its IP adresse

**Reply by riffnshred** [reply]
I just read the documentation for the img. The IP that starts with 10.0 is for your phone when you setup the Wifi on your scoreboard. you need to find your IP address , that is the First Boot / Network Setup  section in the documentation of the img.

https://github.com/falkyre/nhl-led-scoreboard-img#first-boot--network-setup

**Reply by WashCaps33** [reply]
Everything keeps showing 10.0.0.168.  Looked in my Gateway as well and this is what I get.  I’m not seeing 192 or anything close to it anywhere 

![A5F8D609-55D6-4ECC-834D-79B99CDFFEA7](https://user-images.githubusercontent.com/80552501/111092034-3e392900-850b-11eb-8f14-723ecd254eee.jpeg)

**Reply by riffnshred** [reply]
This is a gateway. you used that to setup your scoreboard to connect to your network. From your PC you can use Advance IP scanner, or from your phone, download Fing to scan your network for your raspberry pi

**Reply by WashCaps33** [reply]
I did find a WAN IP address that starts with 75.  Not it?

**Reply by WashCaps33** [reply]
Ok so...just dug around a bit and found this.  I have a modem that is the network I have connected to the Pi.  I also have a dual band router that shows the IP address starting with 169.  So I assume I need to change the Pi settings to connect to the 2.4 band on the router?

**Reply by pjockey** [reply]
You're not behind a VPN or something? If it is 169, then you should probably reset/reboot your router. My Pi is connected to a dual band and it connects just fine, 2.4 or 5, whichever it picks.

**Reply by WashCaps33** [reply]
Nope.  No VPN.  I thought I read that it was not recommended using the 5ghz for this project.  Am I mistaken?

**Reply by WashCaps33** [reply]
And slightly off subject.  Was there any update where it can display and cycle games scheduled for a given day? I saw that it was requested.

**Reply by riffnshred** [reply]
Not for a given day but the current day yes. Its the scoreticker. Check out the wiki section for all the features

**Reply by WashCaps33** [reply]
Awesome.  Found that and set it up.  So damn cool.  Other than the damn Supervisor issue and not being able to get my power button to work, it’s right on point.

**Reply by riffnshred** [reply]
Glad you are up and running man. Enjoy

**Reply by WashCaps33** [reply]
Stupid question, so please forgive me.  But back earlier you said to isolate the bonnet on the newer pi models....what does that mean??

**Reply by riffnshred** [reply]
So if you use the bonnet on any other model than the zero, it will touch other components underneath, (or at least, there is a good chance). To prevent any short and damage to you raapberry pi, I put some electrical tape underneath the bonnet.. if you plan to do the antiflicker mod, I recommend doing that first as you will need to remove the tape to do that operation

**Reply by WashCaps33** [reply]
Thanks for all the help you and pjockey gave me.  Very much appreciated.

**Comment by pjockey** [comment]
Use this site https://github.com/riffnshred/nhl-led-scoreboard/wiki/Step-by-step-guide. to get going. More up to date.

What do you mean you re-formatted the sd card?

**Reply by WashCaps33** [reply]
Dumped out the Raspian Buster Lite I had.   Flashing it onto it again.
